brayns/io/MolecularSystemReader.cpp

```diff
@@ -38,6 +38,10 @@ MolecularSystemReader::MolecularSystemReader(
     , _geometryParameters(geometryParameters)
 {
 }
+std::set<std::string> MolecularSystemReader::getSupportedDataTypes()
+{
+    return {};
```

Comment:
Empty?
I'm looking at the commits one at a time, so maybe it was addressed later.